packages/ffe-buttons/less/base-button.less

```diff
@@ -164,10 +269,17 @@
     border: solid 2px @ffe-grey-silver;
     box-shadow: 0 1px 2px 1px rgba(0, 0, 0, 0.05);
     color: @ffe-blue-azure;
-
-    &.ffe-button--dark {
```

Comment:
Vi trenger ikke denne lenger? Darkmode > dark theme?

Reply:
Bankinterne applikasjoner hos forsikring kan bruke en dark-prop for å legge på sin egen, mørkere stil på et utvalg komponenter. Har ingenting med dark mode å gjøre, så bare å overse. Men de arver dessverre utilsiktet fra sine dark mode moder-komponenter, og derfor må jeg skrive noen ekstra linjer for at de skal se "normale" ut i styleguidist i dark mode.